src/lvm.c

```diff
@@ -1169,12 +1169,14 @@ void luaV_finishOp (lua_State *L) {
 
 /* fetch an instruction and prepare its execution */
 #ifdef USE_YK
+#define NOOPT_VAL(X) asm volatile("" : "+r,m"(X) : : "memory");
 // Elide instruction lookup.
 //
 // FIXME: return type should be `Instruction` not `uint64_t`. We only do this
 // because idempotent support isn't yet implemented for 32-bit return values.
 __attribute__((yk_idempotent))
-uint64_t yk_load_inst(const Instruction *pc) {
+uint64_t load_inst(__attribute__((unused)) uint64_t pv, const Instruction *pc) {
```

Comment:
Better remove `unused` because it now is, and that might confuse us and/or the compiler.

Reply:
Fixed in 3af3722.